### PR TITLE
#164798160    Setting up Sass css

### PR DESCRIPTION
**Pull request by felixkiryowa**
#### What does this PR do?

-  Setting up  Sass 

#### Description of Task to be completed?
-  Setting up Sass to be used to write CSS for elements in the application

#### How should this be manually tested?

- Fetch this branch `git fetch origin ch-setup-sass-164798160`
- Checkout to the branch `git checkout ch-setup-sass-164798160`
- Run `npm install`
- Open `style.scss` file in the `scss` folder at the root of the project.
- Write scss styles` to HTML elements in that file.
- Run `npm run scss` command to compile sccs to css stored in the style.css file in the assets folder at the root of the project.
- Run `npm start`
#### What are the relevant pivotal tracker stories?
[#164798160](https://www.pivotaltracker.com/story/show/164798160)

#### Screen shots
Screenshot 2019-04-30 at 19 22 15

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,4 +2,5 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './components/App';
 
+
```
Comment (dannylwe) [inline src/index.js]: @felixkiryowa this line is not necessary. Kindly revert

File: scss/style.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+$heading: red;
+
+//h1 heading color
+h1 {
+  color: $heading;
+}
```
Comment (dannylwe) [inline scss/style.scss]: Insert a final newline here and in the rest of the files. You can also enable this option in VSCode by adding "files.insertFinalNewline": true to your settings.json file

File: public/dest/style.css [hunk 3]
```diff
@@ -0,0 +1,2 @@
+h1 {
+  color: #333; }
```
Comment (dannylwe) [inline public/dest/style.css]: @felixkiryowa kindly move the curly bracket to a new line

File: assets/css/style.css [hunk 4]
```diff
@@ -0,0 +1,2 @@
+h1 {
+  color: red; }
```
Comment (dannylwe) [inline assets/css/style.css]: @felixkiryowa kindly move the curly bracket to a new line

**Review by dannylwe — CHANGES_REQUESTED** [review]

File: package.json [hunk 5]
```diff
@@ -9,7 +9,8 @@
     "lint": "eslint src --fix",
     "test": "jest",
     "test:watch": "jest --watch",
-    "test:coverage": "jest --collectCoverage"
+    "test:coverage": "jest --collectCoverage",
+    "scss": "node-sass scss/style.scss assets/css/style.css --watch"
```
Comment (adesege) [inline package.json]: You should configure webpack to handle this for you instead of running a script manually

File: public/dest/style.css (hunk 3 above)
Comment (adesege) [inline public/dest/style.css]: You should not commit this file.

**Review by adesege — CHANGES_REQUESTED** [review]

File: assets/css/style.css [hunk 6]
```diff
@@ -0,0 +1,2 @@
+h1 {
```
Comment (adesege) [inline assets/css/style.css]: Remove this file pls

File: jest.config.js [hunk 7]
```diff
@@ -15,6 +15,12 @@ module.exports = {
   // An array of file extensions your modules use
   moduleFileExtensions: ['js', 'json', 'jsx'],
 
+  // A map from regular expressions to module names that allow to stub out resources 
+  // with a single module
+  moduleNameMapper: {
+    '\\.(css|scss|jpg|png)$': '<rootDir>/ignore-css.js',
```
Comment (adesege) [inline jest.config.js]: You can use `ignore-module.js` instead since the file handles image files as well

**Review by adesege — CHANGES_REQUESTED** [review]

**Review by adesege — APPROVED** [review]

**Pull request merged**